pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -217,16 +217,19 @@ func newReconciler(ofClient openflow.Client,
 	idAllocator *idAllocator,
 	fqdnController *fqdnController,
 	groupCounters []proxytypes.GroupCounter,
+	antreaPolicyEnabled bool,
 ) *reconciler {
 	priorityAssigners := map[uint8]*tablePriorityAssigner{}
 	for _, table := range openflow.GetAntreaPolicyBaselineTierTables() {
```

Comment:
If antreaPolicy is not enabled, there's no need to initialize a priorityAssigner for baseline Tier as well

Reply:
Thanks.